android/quest/src/main/assets/configs/app/translations/strings_config.properties

```diff
@@ -16,4 +16,5 @@ add.new.household=Add New Household
 reports=Reports
 transfer.data=Transfer Data
 profile=Profile
-add.family=Add Family
\ No newline at end of file
+add.family=Add Family
+maps=Map
```

Comment:
Since the value is _map_(singular) the key should also match

Reply:
Done